.github/workflows/cross-release.yml

```
@@ -1,80 +1,230 @@
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: ./mapping-v${{ steps.meta.outputs.version }}-c${{ steps.meta.outputs.code }}.txt
+          asset_name: mapping-v${{ steps.meta.outputs.version }}-c${{ steps.meta.outputs.code }}.txt
+          asset_content_type: text/plain
+
 
   # Optional: Signed release builds (requires secrets)
   # (Optional) Add a signed release job later when keystore secrets are ready.
 
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)